api-1.10/src/main/java/org/openmrs/module/reporting/cohort/definition/DrugOrderCohortDefinition.java

```diff
@@ -0,0 +1,192 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.reporting.cohort.definition;
+
+import org.openmrs.Concept;
+import org.openmrs.CareSetting;
+import org.openmrs.Drug;
+
+import org.openmrs.module.reporting.common.Option;
+import org.openmrs.module.reporting.common.Localized;
+import org.openmrs.module.reporting.definition.configuration.ConfigurationProperty;
+import org.openmrs.module.reporting.definition.configuration.ConfigurationPropertyCachingStrategy;
+import org.openmrs.module.reporting.evaluation.caching.Caching;
+
+import java.util.Date;
+import java.util.List;
+
+@Caching(strategy = ConfigurationPropertyCachingStrategy.class)
+@Localized("reporting.DrugOrderCohortDefinition")
+public class DrugOrderCohortDefinition extends BaseCohortDefinition {
+
+    public static final long serialVersionUID = 1L;
+
+    @ConfigurationProperty(group = "which")
+    private Option which;
+
+    @ConfigurationProperty(value = "drugConcepts")
+    private List<Concept> drugConcepts;
+
+    @ConfigurationProperty(value = "drugSets")
+    private List<Concept> drugSets;
+
+    @ConfigurationProperty(value = "activatedOnOrBefore")
+    private Date activatedOnOrBefore;
+
+    @ConfigurationProperty(value = "activatedOnOrAfter")
+    private Date activatedOnOrAfter;
+
+    @ConfigurationProperty(value = "activeOnOrBefore")
+    private Date activeOnOrBefore;
+
+    @ConfigurationProperty(value = "activeOnOrAfter")
+    private Date activeOnOrAfter;
+
+    @ConfigurationProperty(value = "activeOnDate")
+    private Date activeOnDate;
+
+    @ConfigurationProperty(value = "careSetting")
+    private CareSetting careSetting;
+
+    @ConfigurationProperty(value = "drug")
+    private Drug drug;    
```

Comment:
I would think this should be a List<Drug>, similar to the List<Concept> properties